alacritty/src/window_context.rs

```diff
@@ -328,6 +332,14 @@ impl WindowContext {
             // Force the display to process any pending display update.
             self.display.process_renderer_update();
 
+            // XXX it seems like on Wayland when creating a new window the buffer of the old window
+            // may lock leading to resize not being honored, so ask for one more frame to draw in
+            // that particular case.
```

Comment:
```suggestion
            // XXX: On Wayland when creating a new window the buffer of the old window
            // may lock leading to resize not being honored, so ask for one more frame to draw in
            // that particular case.
```

We should be certain, not guessing.